db-migrations/8_add_survey_mode_column.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE survey.survey ADD COLUMN IF NOT EXISTS surveymode character varying(20) NOT NULL;
+
+INSERT INTO survey.survey(surveymode) VALUES('eQ') WHERE shortname IN ('RSI', 'MWSS', 'QBS','MBS');
```

Comment:
Minor point but I think we should just have it be 'EQ' to save having to remember what letter is a capital and what isn't